toqito/state_props/is_abs_ppt.py

```diff
@@ -0,0 +1,128 @@
+"""Check if a quantum state is absolutely PPT."""
+
+import numpy as np
+
+from toqito.matrix_props.is_positive_semidefinite import is_positive_semidefinite
+from toqito.state_opt.symmetric_extension_hierarchy import symmetric_extension_hierarchy
+from toqito.state_props.is_ppt import is_ppt
+from toqito.state_props.is_separable import is_separable
+
+
+def is_abs_ppt(rho: np.ndarray, dim: list[int] = None, max_constraints: int = 2612) -> int:

```

Comment:
Let's slightly modify the return signature of this function such that it returns a value of type `bool | None` as opposed to one of type `int`. In the case where it is absolutely PPT, we would return `True`, not absolutely PPT return `False`, and inconclusive, return `None`.